docs/docs/glossary/continuous-deployment.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Continuous Deployment
+disableTableOfContents: true
+---
+
+## What is continuous deployment?
+
+Continuous deployment is the automation of code deployments. In a continuous deployment system, you don't push a <q>Deploy</q> button or run a `deploy` command. Instead, you build a _pipeline_ &mdash; a process that builds and releases code automatically, without human intervention.
+
+You'll most likely use a service to create your continuous deployment pipeline. Services such as [Netlify](http://netlify.com/), [AWS Amplify](https://aws.amazon.com/amplify/), [Azure](https://azure.microsoft.com/en-us/), and [Zeit](https://zeit.co/) are popular with Gatsby users. Or you can use [Gatsby Builds](https://www.gatsbyjs.org/blog/2020-01-27-announcing-gatsby-builds-and-reports/), a feature of the [Gatsby Cloud](https://www.gatsbyjs.com/) service.
```

Comment:
```suggestion
You'll most likely use a service to create your continuous deployment pipeline. Services such as [Netlify](http://netlify.com/), [AWS Amplify](https://aws.amazon.com/amplify/), [Azure](https://azure.microsoft.com/en-us/), and [Zeit](https://zeit.co/) are popular with Gatsby users. Or you can use [Gatsby Builds](/blog/2020-01-27-announcing-gatsby-builds-and-reports/), a feature of the [Gatsby Cloud](https://www.gatsbyjs.com/) service.
```

I feel like I might be overthinking the order of Gatsby Builds vs other services... "Or you can use Gatsby Builds" feels like maybe it should be considered _after_ all these other very popular options but it may not matter. @hashimwarren?

Reply:
🤔 I think the order and presentation is ok here